home/templates/dashboard/index.html

```diff
@@ -113,7 +113,7 @@ <h6 class="">
 
                             <p class="card-text">
                                 <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Mentor">@{{ issue.mentor }}</a>
-                                <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Level">{{ issue.get_level_display }}</a>
+                                <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Level" style="background-color: {{ issue.get_levelcolor_display }} !important;">{{ issue.get_level_display }}</a>
```

Comment:
Is it absolutely necessary to use important here? If can be avoided, please do. 
Inline styles should generally override other styles

Reply:
Without important, the default color is being considered.